### PR TITLE
Update trading api docs

### PR DESCRIPTION
**Pull request by kenho811-b1**

### CODE REVIEW

File: src/trading-api/next/v1/bullish-trading-api.yml [hunk 1]
```diff
@@ -20,60 +20,61 @@ info:
...
+    - May - add `events` topic to private data WebSocket
+    - April - add market data WebSocket API
+    - April - add private data WebSocket API
+    - March - add optional `depth` parameter to `GET /markets/{symbol}/orderbook/hybrid?depth=10`
+    - March - add IOC time-in-force order type
+    - March - add FOK time-in-force order type
+    - March - add order `statusReasonCode` map to API documentation
+    - March - add historical anonymous trades API - `GET /history/markets/{symbol}/trades`
+    - February - add filter by status=CANCELLED to `GET /orders?status=CANCELLED`
+    - January - add pagination support to `GET /markets/{symbol}/candle`
```
Comment (kenho811-b1) [inline src/trading-api/next/v1/bullish-trading-api.yml:77]: Inverted the order of the changelog. Start from the latest ones at the top.

File: src/trading-api/next/v2/bullish-trading-api.yml [hunk 2]
```diff
@@ -13,68 +13,68 @@ servers:
...
+    - May - add `events` topic to private data WebSocket
+    - April - add market data WebSocket API
+    - April - add private data WebSocket API
+    - March - add optional `depth` parameter to `GET /markets/{symbol}/orderbook/hybrid?depth=10`
+    - March - add IOC time-in-force order type
+    - March - add FOK time-in-force order type
+    - March - add order `statusReasonCode` map to API documentation
+    - March - add historical anonymous trades API - `GET /history/markets/{symbol}/trades`
+    - February - add filter by status=CANCELLED to `GET /orders?status=CANCELLED`
+    - January - add pagination support to `GET /markets/{symbol}/candle`
```
Comment (kenho811-b1) [inline src/trading-api/next/v2/bullish-trading-api.yml:76]: Inverted the order of the changelog. Start from the latest ones at the top.

File: src/trading-api/next/v2/bullish-trading-api.yml [hunk 3]
```diff
@@ -13,68 +13,68 @@ servers:
...
     - follows [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) conventions
-    - endpoints fall under one of the two categories:
-      - Authenticated, they provide access to private client data e.g. [Get Orders](#get-/orders) endpoint
-      - Non-authenticated, they provide access to public data e.g. [Get Markets](#get-/markets) endpoint
-    - uses bearer based authentication
+    - has the base URL `api.exchange.bullish.com` (unless one of the sandbox environments is being used)
+    - has endpoints under the below categories:
+      - Authenticated endpoints for private client data e.g. [Get Orders](#get-/orders) endpoint
+      - Non-authenticated endpoints for public data e.g. [Get Markets](#get-/markets) endpoint
+    - uses bearer-based authentication
```
Comment (kenho811-b1) [inline src/trading-api/next/v2/bullish-trading-api.yml:26]: all lines start with a **verb**

File: src/trading-api/next/v1/bullish-trading-api.yml [hunk 4]
```diff
@@ -20,60 +20,61 @@ info:
...
     - follows [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) conventions
-    - endpoints fall under one of the two categories:
-      - Authenticated, they provide access to private client data e.g. [Get Orders](#get-/orders) endpoint
-      - Non-authenticated, they provide access to public data e.g. [Get Markets](#get-/markets) endpoint
-    - uses bearer based authentication
+    - has the base URL `api.exchange.bullish.com` (unless one of the sandbox environments is being used)
+    - has endpoints under the below categories:
+      - Authenticated endpoints for private client data e.g. [Get Orders](#get-/orders) endpoint
+      - Non-authenticated endpoints for public data e.g. [Get Markets](#get-/markets) endpoint
+    - uses bearer-based authentication
```
Comment (kenho811-b1) [inline src/trading-api/next/v1/bullish-trading-api.yml:27]: all lines start with a **verb**.

File: src/trading-api/next/v1/bullish-trading-api.yml [hunk 5]
```diff
@@ -85,9 +86,9 @@ info:
 
     There are 4 special query parameters used to control the pagination behaviour.
     - `_pageSize` can be one of `5, 25, 50, 100`, default value is `25`
-    - `_nextPage` cursor to the next page
-    - `_previousPage` cursor to the previous page
     - `_metaData` can be false/true, if false the links are NOT part of the response. explicitly set `_metaData=true` to guarantee links are returned.
+    - `_nextPage` cursor to the next page. It is provided in the paginated response when `_metaData=true`.
+    - `_previousPage` cursor to the previous page. It is provided in the paginated response when `_metaData=true`.
```
Comment (kenho811-b1) [inline src/trading-api/next/v1/bullish-trading-api.yml:91]: highlights that the value to _nextPage and _previousPage are provided in the response where _metaData=true

File: src/trading-api/next/v1/bullish-trading-api.yml [hunk 6]
```diff
@@ -153,24 +166,39 @@ info:
...
-
     ## Increasing Rate Limits
+    
+    Bullish has a tiered rate limit offering based on volume. 
+    
+    |  | Default | 30-Day Volume between $1B USD and $2B USD | 30-Day Volume over $2B USD |
+    | ---------- |---------------- |---------------- |---------------- |
+    | Cloudflare | 50 msgs/sec | same as default | same as default
+    | Cloudflare Bypass | 100 msgs/sec  | up to 200 msgs/sec | up to 500 msgs/sec
+
```
Comment (kenho811-b1) [inline src/trading-api/next/v1/bullish-trading-api.yml:177]: Details w.r.t. increasing rate limits.

image (268)

File: src/trading-api/next/v2/bullish-trading-api.yml [hunk 7]
```diff
@@ -2479,6 +2515,24 @@ paths:
...
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Asset'
+        429:
+          description: Too Many Requests
+        500:
+          description: Internal Server Error
```
Comment (kenho811-b1) [inline src/trading-api/next/v2/bullish-trading-api.yml:2535]: **added Get Asset by Symbol**

image

File: src/trading-api/next/v1/bullish-trading-api.yml [hunk 8]
```diff
@@ -2357,6 +2393,24 @@ paths:
...
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Asset'
+        429:
+          description: Too Many Requests
+        500:
+          description: Internal Server Error
```
Comment (kenho811-b1) [inline src/trading-api/next/v1/bullish-trading-api.yml:2413]: **added Get Asset by Symbol**

image

File: src/trading-api/next/v1/bullish-trading-api.yml [hunk 9]
```diff
@@ -499,6 +529,12 @@ info:
     2. Sign the hexdigest from step 1 with ECDSA P-256 using your `<PRIVATE_KEY>`
     3. Various signing examples can be found on [GitHub](https://github.com/bullish-exchange)
 
+    ## Fetch Trading Account Ids
+    
+    Trading Account Ids may be required by some endpoints. They can be fetched from [Get All Trading Accounts](#get-/accounts/trading-accounts).
```
Comment (kenho811-b1) [inline src/trading-api/next/v1/bullish-trading-api.yml:534]: added a section to fetch trading account ids.

image

File: src/trading-api/next/v2/bullish-trading-api.yml [hunk 10]
```diff
@@ -552,6 +584,10 @@ info:
     2. Sign the hexdigest from step 1 with ECDSA P-256 using your `<PRIVATE_KEY>`
     3. Various signing examples can be found on [GitHub](https://github.com/bullish-exchange)
 
+    ## Fetch Trading Account Ids
+    
+    Trading Account Ids may be required by some endpoints. They can be fetched from [Get All Trading Accounts](#get-/accounts/trading-accounts).
```
Comment (kenho811-b1) [inline src/trading-api/next/v2/bullish-trading-api.yml:589]: added a section to fetch trading account ids.

image

File: src/trading-api/next/v1/bullish-trading-api.yml [hunk 11]
```diff
@@ -153,24 +166,39 @@ info:
...
+    To express a quantity of 1 BTC in `BTCUSD` market, therefore, it has to be `1.00000000` BTC.
+    
+    Alternatively, the below market APIs can also be queried to obtain the price and quantity precisions for different markets.
+    
+    - The [Get Markets](#get-/markets) API endpoint for all supported market symbols
+    - The [Get Market By Symbol](#get-/markets/-symbol-) API endpoint for a specific market symbol
+    
+    Following the logic outlined above, 
     - The precision for quantity is defined by the `basePrecision`
     - The precision for price is defined by `quotePrecision`
```
Comment (kenho811-b1) [inline src/trading-api/next/v1/bullish-trading-api.yml:203]: included endpoints for /assets as well

**Review by sandeeprakhrablockone — APPROVED** [review]
👌

**Pull request merged**